Header header
float32[] leftDistance #array size 360 of leftIR in cm
float32[] rightDistance#                  rightIR
float32 initialAngle   #Initial angle so that we can set initialAngle+(index of relevant dist.)
